examples/fungible-token/ft/src/lib.rs

```diff
@@ -18,7 +18,10 @@ NOTES:
 use near_contract_standards::fungible_token::metadata::{
     FungibleTokenMetadata, FungibleTokenMetadataProvider, FT_METADATA_SPEC,
 };
-use near_contract_standards::fungible_token::FungibleToken;
+use near_contract_standards::fungible_token::{
+    FungibleToken, FungibleTokenCore, FungibleTokenResolver, StorageManagement,
+};
+use near_contract_standards::storage_management::{StorageBalance, StorageBalanceBounds};
```

Comment:
Given that `StorageBalance` and `StorageBalanceBounds` are imported from storage_management, let's import `StorageManagement` from there as well, instead of re-exporting it through `fungible_token`.